crates/runtime/src/continuation.rs

```diff
@@ -207,6 +208,20 @@ pub fn resume(
         }
     }
 
+    unsafe {
+        let runtime_limits = &**instance.runtime_limits();
+
+        (*parent_stack_limits).stack_limit = *runtime_limits.stack_limit.get();
+        (*parent_stack_limits).last_wasm_entry_sp = *runtime_limits.last_wasm_entry_sp.get();
+        // These last two values were only just updated in the `runtime_limits`
+        // because we entered the current libcall.
+        (*parent_stack_limits).last_wasm_exit_fp = *runtime_limits.last_wasm_exit_fp.get();
+        (*parent_stack_limits).last_wasm_exit_pc = *runtime_limits.last_wasm_exit_pc.get();
+
+        *runtime_limits.stack_limit.get() = (*contobj).limits.stack_limit;
+        *runtime_limits.last_wasm_entry_sp.get() = (*contobj).limits.last_wasm_entry_sp;
```

Comment:
Isn't this zero on the initial invocation? If so, isn't that a bug?

Reply:
Yes and no. For a freshly created continuation, its `limits.last_wasm_entry_sp` is indeed 0 (or any other unspecific placeholder for it not being set before). But immediately after these instructions run, we resume the continuation. Because the continuation hasn't run before, this means executing its array call trampoline, which in turn sets the `last_wasm_entry_sp` in the `VMRuntimeLimits` to the correct value.
This behaviour is documented [here](https://github.com/wasmfx/wasmfxtime/pull/99/files#diff-31ba266d671a4acbdb0c7134ea9f783bab8922bd74203d5e97b94fc7853adb96R122)

Reply:
Thanks for clarifying. I think it would be good to put a comment on line 220 with a back reference to the top comment.